index.html

```diff
@@ -950,66 +953,68 @@ <h2>Types</h2>
 
         <p>
 All <a>credentials</a>, <a>presentations</a>, and encapsulated objects MUST
-specify or be associated with additional, more narrow <a>type</a>s
-(e.g., <code>ProofOfAgeCredential</code>) such that software systems can use
-the additional information to more easily process the data.
-        </p>
-
-        <p>
-When processing encapsulated objects in this specification, (e.g., objects
-associated with the <code>credentialSubject</code> property or deeply nested therein),
-a software system SHOULD use type information specified in encapsulating
-objects higher in the hierarchy. For the avoidance of doubt, an
-encapsulating object such as <a>credential</a>, SHOULD convey the types of
-associated objects so that the <a>verifier</a> can quickly determine the
-contents of the associated object based on the type of the encapsulating object.
-To provide a concrete example, a <a>credential</a> with the additional
-<code>type</code> of <code>ProofOfAgeCredential</code> would signal to the
-<a>verifier</a> that the object associated with the <a>credentialSubject</a> property will
-contain the identifier for the <a>subject</a> in the <code>id</code> property
-and the age assertion in the <code>ageOver</code> property. This enables
-implementers to rely on values associated with the <code>type</code> property
-for verification purposes. The expectation of types and their associated
-properties SHOULD be documented in at least a human-readable specification
-and, preferably, in an additional machine-readable representation.
+specify, or be associated with, additional more narrow <a>type</a>s (like 
+<code>ProofOfAgeCredential</code>, for example) so software systems can 
+process this additional information.
+        </p>
+
+        <p>
+When processing encapsulated objects defined in this specification, (for 
+example, objects associated with the <code>credentialSubject</code> object 
+or deeply nested therein), software systems SHOULD use the <a>type</a> 
+information specified in encapsulating objects higher in the hierarchy. 
+Specifically, an encapsulating object, such as <a>credential</a>, SHOULD 
+convey the associated object <a>types</a> so that <a>verifiers</a> can quickly 
```

Comment:
This `types` link and the one below are not showing up (and the commit diff previewer seems confused by them).  See the `dfn` comment for a fix.

Reply:
Great suggestion!